src/service/templates/service.json.js

```diff
@@ -2,11 +2,13 @@ module.exports = function(config) {
   return {
     require: config.require,
     options: {
-      paginate: 'config.paginate',
-      Model: config.model
+      Model: `./${config.name}.model` ,
+      options: [config.model || {}]
```

Comment:
I'm thinking this should probably be tweaked so if you just pass an adapter type we can point it at a template / default instantiation of that model, but again, I'll give this some breathing room until we get better alignment with how the model generator works so we aren't duplicating templates